helpers/colour-contrast/accessibility.js

```diff
@@ -0,0 +1,17 @@
+import { contrastColors, contrastRGB } from './contrast.js';
+
+const contrastMin = 4.5;
+
+export const contrastNormalText = contrastMin;
+export const contrastLargeTxt = 3;
+export const contrastNonText = 3;
+
+// takes 2 color strings
+export function isAccessible(color1, color2, myContrastMin) {
+	return contrastColors(color1, color2) >= (myContrastMin || contrastMin);
+}
```

Comment:
Is there still a difference between `isAccessible` and `isAccessibleRGB` now that it only supports RGB colours?

Reply:
Oh, I see, the second one uses the RGB object -- never mind! Not sure if we need to export the second one though (or even have it?), as I'm not sure folks will be creating those objects?